docs/sources/mimir/migration-guide/migrating-from-cortex.md

```diff
@@ -316,8 +286,7 @@ You can update to the Grafana Mimir Helm chart from the Cortex Helm chart.
    nameOverride: "cortex"
    ```
 
-   h. Disable MinIO.
-   The Grafana Mimir Helm chart enables MinIO by default for convenience during first time installs.
+   g. Disable MinIO.
    If you are migrating from Cortex, use your existing object storage, and disable MinIO.
```

Comment:
```suggestion
```

nit: Is this what the whole article is about?

Reply:
but if you decide to drop line 290, then I think the previous line may give some context as to why they have to disable minio - "The Grafana Mimir Helm chart enables MinIO by default for convenience during first time installs." So maybe keep that?

Reply:
Yes make sense. I am editing the paragraph to give more context on why we want them to disable minio.